libraries/provider_database_postgresql.rb

```diff
@@ -100,10 +100,12 @@ def exists?
           ret
         end
 
-        # Test if text is psql keyword
-        def keyword?(text)
+        # Verify the postgres server's version number is greater than the integer passed in
+        def version_greater_than?(desired_version_int)
           begin
-            result = db('template1').exec_params('select * from pg_get_keywords() where word = $1', [text.downcase]).num_tuples != 0
+            ret = db('template1').exec('SHOW server_version_num;')
+            server_version_int = ret.getvalue(0,0).to_i
+            result = server_version_int > desired_version_int
```

Comment:
I don't see why you feel the need to assign the result of the evaluation to a variable here, it goes out of scope immediately afterwards.